ui/spinner.js

```diff
@@ -203,20 +208,31 @@ return $.widget( "ui.spinner", {
 
 	_draw: function() {
 		var uiSpinner = this.uiSpinner = this.element
-			.addClass( "ui-spinner-input" )
 			.attr( "autocomplete", "off" )
 			.wrap( this._uiSpinnerHtml() )
 			.parent()
 				// add buttons
 				.append( this._buttonHtml() );
 
+		this._addClass( this.uiSpinner, "ui-spinner", "ui-widget ui-widget-content" );
+		this._addClass( "ui-spinner-input" );
+
 		this.element.attr( "role", "spinbutton" );
 
 		// button bindings
-		this.buttons = uiSpinner.find( ".ui-spinner-button" )
+		this.buttons = uiSpinner.find( "a" )
```

Comment:
Oh, I see, because the `_*Html()` methods were changed. I'm not sure this keeps the intention of those methods. Anyway, it no longer seems useful to keep those methods with the markup being reduced so much.